Antonio_Ramos_Documentation

```diff
@@ -352,3 +352,20 @@ RuntimeError: b'no arguments in initialization list'"
 - opened spyder and ran program "K-function Local_3Stage.py" got infinite simulation error loop again. 
 
 - will repeat all previous steps from today's documentation on my personal computer to see if infinite loop simulation error occurs 
+
+7/27/2020
+- reviewing data in "DEC_00_SF3_DP3_with_ann.csv" and noticed that a few cells with 0 manufacturing employment have ZCTAs with letters "XX" and "HH" in them. will do a find a replece to remove all alphabetic letters from Id2 column and delete all data columns not being utilized by 3stage program. Saved resulting csv file as "DEC_00_SP3_DP3_with_ann2.csv". Note. this changes the data type of the columns into objects must revert types to integer and strings in python script 
```

Comment:
Now it makes sense why it didn't want to be numeric!

Could the find and replace you did (I'm assuming in Excel) be done programmatically in Python before converting the data type? If so, please convert the process to do this--it would make our work more reproducible.